cobra/solvers/cplex_solver.py

```diff
@@ -93,7 +93,7 @@ def format_solution(lp, cobra_model, **kwargs):
         x = y = x_dict = y_dict = objective_value = None
 
     return Solution(objective_value, x=x, x_dict=x_dict, status=status,
-                    y=y, y_dict=y_dict)
+                    y=y, y_dict=y_dict, solver=solver_name)
```

Comment:
Sure, I can do that. Honestly I thought I would have needed it to do this PR, but then ended up not needing it.